app-android/src/main/java/com/ustadmobile/port/android/view/UstadBaseFragment.kt

```diff
@@ -36,6 +44,19 @@ open class UstadBaseFragment : Fragment(), UstadView, DIAware {
         }
 
 
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setHasOptionsMenu(true)
+    }
+
+    override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
+        super.onCreateOptionsMenu(menu, inflater)
+        searchView = menu.findItem(R.id.menu_search).actionView as SearchView
+        searchManager = SearchViewManagerLifecycleObserver(searchView).also {
```

Comment:
The creation of the SearchViewManagerLifecycleObserver should be done in onCreateView or onViewCreated. This is effectively recreating the manager every time the options menu is invalidated.